crates/autopilot/src/database/fee_policies.rs

```diff
@@ -84,19 +90,36 @@ mod tests {
             surplus_factor: None,
             max_volume_factor: None,
             volume_factor: Some(0.06),
+            quote_sell_amount: None,
+            quote_buy_amount: None,
+        };
+        // price improvement fee policy
+        let fee_policy_4 = dto::FeePolicy {
+            auction_id,
+            order_uid,
+            kind: dto::fee_policy::FeePolicyKind::PriceImprovement,
+            surplus_factor: Some(0.3),
```

Comment:
I really don't have a good feeling about overloading columns from other variants (why the hell is the price improvement factor stored in the column called surplus factor)? Let's treat ourselves to a new column or at least rename this to something suggesting that this is just "a random factor that may be reused for any fee policy kind".